quickwit/quickwit-serve/src/lib.rs

```diff
@@ -439,6 +434,8 @@ pub async fn serve_quickwit(
     shutdown_signal: BoxFutureInfaillible<()>,
     env_filter_reload_fn: EnvFilterReloadFn,
 ) -> anyhow::Result<HashMap<String, ActorExitStatus>> {
+    let _ = rustls::crypto::ring::default_provider().install_default();
```

Comment:
Let's move that up the call stack, in `cli/main`, where we init all the other singletons.